topics/admin/tutorials/ansible-galaxy/tests/1-galaxy-up.sh

```diff
@@ -1,2 +1,2 @@
 # assumes letsencript staging certificate is preconfigured on VM
-galaxy-wait --timeout 30 -g https://$(hostname -f) -v
+galaxy-wait --timeout 30 -g https://"${GALAXY_HOSTNAME}" -v
```

Comment:
note: this line has just been changed a minute ago (https://github.com/galaxyproject/training-material/pull/2609/files#diff-5d2156894e9f199788fbb4d7e42ea35a3bc3852939bcf3d8215907df6c2c877aR2)